### ISSUE TITLE
iou issue

### ISSUE THREAD

**Issue by yyfyan** [Zzh-tju/DIoU-SSD-pytorch#5]
Why use normalized(eg,0-1) x1,y1,x2,y2 to calculate IOU ?

**Comment by yyfyan** [comment]
@Zzh-tju

**Comment by yyfyan** [comment]
and I find you use :
loss = weight * loss_l + loss_c
the weight default is 5.

**Comment by Zzh-tju** [comment]
Normalizing all the coordinates to [0,1] will benefit the bbox regression, otherwise they're large numbers which is not good for  network training.

The original loss function is smoothL1 which is almost five times of the IoU based loss.
